spec/rspec/rails/configuration_spec.rb

```diff
@@ -291,6 +291,7 @@ def in_inferring_type_from_location_environment
   end
 
   it "metadata `type: :request` sets up request example groups" do
+    require "rspec/rails/example/request_example_group"
```

Comment:
I was going to comment "This could cause order dependent failures, we typically do such requires in a sub process to avoid posioning the env" but its made me think prehaps we should be doing that all over now if we lazy load so that we don't accidentally depend on a module load order that we just happen to have in our specs 😧